pkg/cmd/roachtest/roachtestflags/flags.go

```diff
@@ -357,6 +357,12 @@ var (
 		Usage: `Use SpotVM to run tests, If the provider does not support spotVM, it will be ignored`,
 	})
 
+	AutokillThreshold float64 = 1.0
+	_                         = registerRunFlag(&AutokillThreshold, FlagInfo{
+		Name:  "autokillThreshold",
+		Usage: `The threshold of a failed tests after which the nightlies will be auto-killed.`,
```

Comment:
nit: the intention is to use this for nightlies but this should be generic enough to work for any roachtest run

Reply:
also, can we use `auto-kill-threshold` to keep the naming consistent?

Reply:
Updated